samcli/local/apigw/local_apigw_service.py

```diff
@@ -438,6 +459,59 @@ def _construct_event(flask_request, port, binary_types, stage_name=None, stage_v
         LOG.debug("Constructed String representation of Event to invoke Lambda. Event: %s", event_str)
         return event_str
 
+    @staticmethod
+    def _construct_event_http(flask_request, port, binary_types, stage_name=None, stage_variables=None, route_key=None):
+        """
+        Helper method that constructs the Event 2.0 to be passed to Lambda
+
+        https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html
+
+        :param request flask_request: Flask Request
+        :return: String representing the event
+        """
+        # pylint: disable-msg=too-many-locals
+
+        endpoint = PathConverter.convert_path_to_api_gateway(flask_request.endpoint)
```

Comment:
I think this unused variable is the only reason for the failing check.

```suggestion
```

Reply:
@jnodorp thanks!